frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -44,7 +44,7 @@ private void showCantGetBranchesMessage(PsiFile file) {
       return;
     }
     HintManager.getInstance().showErrorHint(currentEditor,
-        "We can't get project branches, so we can't checking it names", HintManager.ABOVE);
+        "We can't get project branches, so we can't complete their names", HintManager.ABOVE);
```

Comment:
I don't like this message. :confused: Who is `we` from IDE? Can you make it impersonal?

Reply:
Whoops right... `we` doesn't sound good in messages like that

Reply:
Passive voice FTW

Reply:
Let me disagree. This kind of user facing messages became more popular (it is visible especially in Microsoft products):
![we_cant](https://user-images.githubusercontent.com/56402877/87034265-dbf39000-c1e7-11ea-9aaf-9a1670b8678c.jpg)
But ok, I can change it to passive voice.

Reply:
Anyway, would be good to keep it consistent with what's prevalent in the rest of the plugin/IntelliJ in general, whatever that happens to be

Reply:
1. consistency
2. `we` is undefined. From the user point of view: Is `we` Jet Brains or Git Machete devs or who?

Reply:
Actually, this message should say that we can't **check** their names (not complete)